lib/atscppapi/src/include/atscppapi/PluginInit.h

```diff
@@ -37,10 +38,10 @@ extern "C" {
  */
 void TSPluginInit(int argc, const char *argv[]);
 
-enum TsReturnCode {
-  TS_ERROR = -1,
-  TS_SUCCESS = 0,
-};
+//enum TsReturnCode {
```

Comment:
Commenting out like this is fine for initial work but should be cleaned up for the pull request.